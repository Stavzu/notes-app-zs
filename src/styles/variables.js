export const colors = {
    silver: '#D9DCE1',
    iron: '#CACDD0',
    boulder: '#7D7878',
    regentGrey: '#949EA8',
    hellGrey: '#242529',
    lightGrey: '#A9AEB4',
    white: '#FFFFFF',
    strawberry: '#FF4081',
    strawberryDarker: '#E73370',
    skyBlue: '#81cfdf',
    black: '#000000',
    greyDark: '#bababa',
    greyMid: '#404040',
    sherpaBlue: '#34495e',
    shark: '#242529',
    ming: '#326e7b'
  }
  