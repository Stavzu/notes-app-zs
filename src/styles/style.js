import React from 'react'
import { createGlobalStyle } from "styled-components"
import CSSReset from  '../ResetCSS'
import * as variables from './variables'

export const GlobalStyle = createGlobalStyle`
  ${CSSReset} 

  *,
  *:before,
  *:after {
      box-sizing: border-box;
  }

  html { 
      font-size: 62.5%; 
      overflow-x: hidden; 
      color: ${variables.colors.hellGrey};
    }

  a { 
      text-decoration: none !important; 
      color: ${variables.colors.hellGrey};
    }

  body { 
      font-size: 1.6rem; 
      margin:0;
      padding:0;
      font-family: 'Roboto', sans-serif;
    } 
  h1  { 
      font-size: 2.4rem; 
    } 


  textarea[type="text"]:hover, 
  input[type="text"]:hover, 
  textarea:active, 
  input[type="text"]:active, 
  textarea:focus, 
  input[type="text"]:focus,
  button:focus,
  button:active,
  button:hover,
  label:focus,
  .btn:active,
  .btn.active
  {
      outline:none;
      -webkit-appearance:none;
  }
`