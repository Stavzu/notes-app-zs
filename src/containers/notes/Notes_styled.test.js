import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import * as variables from '../../styles/variables'

const Result = styled.div`
margin-top: 1rem;
`
const BackText = styled.span`
margin-left: 1rem;
font-size: 1.2rem;
font-weight: 500;
`
const Title = styled.h1`
font-size: 2rem;
padding: .5rem;
text-align: center;
color: ${variables.colors.skyBlue};
`
const UpdatedWrap = styled.div`
font-size: 1.4rem;
margin-top: 2.5rem;
text-align: center;
`
const Here = styled.button`
margin-top: 1rem;
padding: .7rem 1.5rem;
border: .1rem solid ${variables.colors.black};
border-radius: .5rem;
cursor: pointer;
outline: none;
`
const BackBtn = styled.button`
padding: .5rem;
margin: 2rem;
border: none;
background: transparent;
outline: none;
cursor: pointer;
`


test('it works result', () => {
  const tree = renderer.create(<Result />).toJSON()
  expect(tree).toMatchSnapshot()
})

test('it works backtext', () => {
    const tree = renderer.create(<BackText />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works title', () => {
    const tree = renderer.create(<Title />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works here', () => {
    const tree = renderer.create(<Here />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works updatedwrap', () => {
    const tree = renderer.create(<UpdatedWrap />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('it works backbtn', () => {
    const tree = renderer.create(<BackBtn />).toJSON()
    expect(tree).toMatchSnapshot()
})