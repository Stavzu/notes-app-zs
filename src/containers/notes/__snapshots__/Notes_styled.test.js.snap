// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works backbtn 1`] = `
.c0 {
  padding: .5rem;
  margin: 2rem;
  border: none;
  background: transparent;
  outline: none;
  cursor: pointer;
}

<button
  className="c0"
/>
`;

exports[`it works backtext 1`] = `
.c0 {
  margin-left: 1rem;
  font-size: 1.2rem;
  font-weight: 500;
}

<span
  className="c0"
/>
`;

exports[`it works here 1`] = `
.c0 {
  margin-top: 1rem;
  padding: .7rem 1.5rem;
  border: .1rem solid #000000;
  border-radius: .5rem;
  cursor: pointer;
  outline: none;
}

<button
  className="c0"
/>
`;

exports[`it works result 1`] = `
.c0 {
  margin-top: 1rem;
}

<div
  className="c0"
/>
`;

exports[`it works title 1`] = `
.c0 {
  font-size: 2rem;
  padding: .5rem;
  text-align: center;
  color: #81cfdf;
}

<h1
  className="c0"
/>
`;

exports[`it works updatedwrap 1`] = `
.c0 {
  font-size: 1.4rem;
  margin-top: 2.5rem;
  text-align: center;
}

<div
  className="c0"
/>
`;
